features/networking/multus.feature

```diff
@@ -677,29 +678,16 @@ Feature: Multus-CNI related scenarios
     the bridge interface named "mybridge.100" is deleted from the "<%= cb.nodes[1].name %>" node
     """  
     #making sure the pods on same node can ping while pods on diff nodes can't
-
     When I execute on the "<%= cb.pod1 %>" pod:
```

Comment:
Too much ping combinations so i reduced to effective ones and the reason it failed sometime was because IPAM range in net-attach-def defined that subnet for each node. Let say range is 22.22.22.10 - 22.22.22.20 then its possible that one pod on a node gets 22.22.22.10 and other pod on new node might get same IP, so if that pod pings first pod, it will actually ping itself and throw a success. I used arping which eliminated the duplicacy issues. Tested again on ovn/sdn clusters

Reply:
Agree, arping will resolve the issue of the duplicate IP address assigned to different pods in different nodes which cause the testing failed sometimes.